.gitattributes

```diff
@@ -0,0 +1,2 @@
+poetry.lock binary linguist-generated
+deployments/**/* binary linguist-generated
```

Comment:
Why are we ignoring everything under `deployments`?
Those files are tracked explicitly.